.github/workflows/maven.yml

```diff
@@ -74,6 +74,9 @@ on:
         default: 'zulu'
         type: string
 
+env:
+  CI: true
+
```

Comment:
it is a default value ... should not be overridden

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables